pkg/ios/dsym-utils.go

```diff
@@ -47,11 +47,10 @@ func FindDsymsInPath(path string) ([]*DwarfInfo, string, error) {
 			} else {
 				log.Info("Unzipped " + fileName + " to " + tempDir + " for uploading")
 				dsymLocations = findDsyms(tempDir)
-
 			}
 
-		} else if strings.HasSuffix(strings.ToLower(path), ".dsym") {
-			// If path points to a .dSYM file, then we will use it as is
+		} else {
+			// If path points to a file, then we will assume it is a dSYM and use it as-is
```

Comment:
I changed this because I thought of an edge case where the path could be a file, but not have the `.dSYM` suffix i.e. the actual file that resides in the `DWARF` directory of a path. So if it is referenced directly, we can still handle it.